assets/js/admin/form-builder.js

```diff
@@ -215,6 +215,7 @@
 			EVFPanelBuilder.bindToggleHandleActions();
 			EVFPanelBuilder.bindLabelEditInputActions();
 			EVFPanelBuilder.bindSyncedInputActions();
+			EVFPanelBuilder.initializeRangeSliderFields();
```

Comment:
This `initializeRangeSliderFields` and all the methods newly introduced should strictly reside in Pro.

Reply:
Resolved with bacf45179fd6783ab70ef3a3b3778e656b75203a .